<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flow_left</class>
 <widget class="QWidget" name="flow_left">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>587</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #e7f5ff;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="flow_name">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000000;
font-size: 16px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>흐름명</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="input_label">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000000;
font-size: 14px</string>
     </property>
     <property name="text">
      <string>입력</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="data_source_input">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>38</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>38</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">                QComboBox {
                            background-color: white;
                            border: 1px solid #dee2e6;
                            border-radius: 4px;
                            padding: 5px 10px;
                            min-height: 30px;
                            color: #495057;
                        }
                        QComboBox:focus {
                            border: 1px solid #339af0;
                        }
                        QComboBox::drop-down {
                            subcontrol-origin: padding;
                            subcontrol-position: center right;
                            width: 30px;
                            border: none;
                        }
                        QComboBox::down-arrow {
                            image: url(src/ui/resources/images/down-arrow-svgrepo-com.svg);
                            width: 10px;
                            height: 10px;
                            margin-right: 1px;
                        }
                        QComboBox QAbstractItemView {
                            background-color: white;
                            border: 1px solid #dee2e6;
                            border-radius: 4px;
                            outline: none;
                            selection-background-color: #e9ecef;
                            selection-color: #000;
                        }
                        QComboBox QAbstractItemView::item {
                            padding: 8px 20px;
                            color: #495057;
                        }
                        QComboBox QAbstractItemView::item:selected {
                            background-color: #e9ecef;
                            color: #000;
                        }</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="process_header">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="process_label">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #000000;
font-size: 14px;</string>
       </property>
       <property name="text">
        <string>처리</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_process_btn">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: none;
    background: transparent;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="process_list">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget {
    background-color: #ffffff;
    color: #495057;
    border: 1px solid #dee2e6;
    border-radius: 4px;
}
QListWidget::item {
    padding: 8px;
}
QListWidget::item:selected {
    background-color: #e7f5ff;
    color: #1864ab;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="result_label">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000000;
font-size: 14px;</string>
     </property>
     <property name="text">
      <string>결과</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="result_text">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #ffffff;
    color: #495057;
    border: 1px solid #dee2e6;
    border-radius: 4px;
    padding: 8px;
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
